can_tools/scrapers/official/WY/WYVaccines.py

```diff
@@ -297,3 +297,184 @@ def normalize(self, data) -> pd.DataFrame:
+    body65UpPercentage = '{"dataRequest":[{"requestContext":{"reportContext":{"reportId":"59351f95-49e8-4440-b752-4d9b919dca88","pageId":"26374700","mode":"VIEW","componentId":"cd-tarmc5tlic","displayType":"simple-barchart"}},"datasetSpec":{"dataset":[{"datasourceId":"f264378a-4e61-41cf-9017-ba25e1a3ba22","revisionNumber":0,"parameterOverrides":[]}],"queryFields":[{"name":"qt_6dfwfes6ic","datasetNs":"d0","tableNs":"t0","dataTransformation":{"sourceFieldName":"_2024258922_"}},{"name":"qt_lhchles6ic","datasetNs":"d0","tableNs":"t0","dataTransformation":{"sourceFieldName":"_1863824869_","aggregation":6}}],"sortData":[{"sortColumn":{"name":"qt_6dfwfes6ic","datasetNs":"d0","tableNs":"t0","dataTransformation":{"sourceFieldName":"_2024258922_"}},"sortDir":0}],"includeRowsCount":true,"paginateInfo":{"startRow":1,"rowsCount":25},"blendConfig":{"blockDatasource":{"datasourceBlock":{"id":"block_ssp17tjijc","type":1,"inputBlockIds":[],"outputBlockIds":[],"fields":[]},"blocks":[{"id":"block_tsp17tjijc","type":5,"inputBlockIds":[],"outputBlockIds":[],"fields":[],"queryBlockConfig":{"joinQueryConfig":{"joinKeys":[],"queries":[{"datasourceId":"f264378a-4e61-41cf-9017-ba25e1a3ba22","concepts":[]}]}}}],"delegatedAccessEnabled":true,"isUnlocked":true,"isCacheable":false}},"filters":[{"filterDefinition":{"filterExpression":{"include":false,"conceptType":0,"concept":{"ns":"t0","name":"qt_cn2odwj1fc"},"filterConditionType":"EQ","stringValues":["Wyoming"],"numberValues":[],"queryTimeTransformation":{"dataTransformation":{"sourceFieldName":"_2024258922_"}}}},"dataSubsetNs":{"datasetNs":"d0","tableNs":"t0","contextNs":"c0"},"version":3}],"features":[],"dateRanges":[],"contextNsCount":1,"calculatedField":[],"needGeocoding":false,"geoFieldMask":[],"geoVertices":100000},"useDataColumn":true}]}'
+
+    state_fips = int(us.states.lookup("Wyoming").fips)
+    execution_dt = pd.Timestamp.now()
+    source = (
+        "https://health.wyo.gov/publichealth/immunization/wyoming-covid-19-vaccine-information/"
+        "covid-19-vaccine-distribution-data/"
+    )
+    source_name = "Wyoming Department of Health"
+    baseUrl = "https://datastudio.google.com/batchedDataV2"
+    resource_id = "a51cf808-9bf3-44a0-bd26-4337aa9f8700"
+    has_location = False
+    location_type = "county"
+
+    def fetch(self):
+        """
+        Pulls Wyoming county vaccine data, cleans and up the Json Strig that is returned before returning that object as
+        a dictionary
+        """
+        WYVacDataTotal = self.get_dataset(
+            json.loads(self.bodyTotalPercentage), url=self.baseUrl
+        )
+        parsedVacDataTotal = json.loads(WYVacDataTotal[10 : len(WYVacDataTotal) - 1])
```

Comment:
I get a json parsing error before changing where the parsing starts by one char (for the other ages too)

```suggestion
        parsedVacDataTotal = json.loads(WYVacDataTotal[11 : len(WYVacDataTotal) - 1])
```